doc/upgrade/7.2.md

```diff
@@ -4,7 +4,7 @@ See `doc/bc/changes-7.2.md` for requirements changes and deprecations.
 
 ## MySQL/MariaDB database tables character set change
 
-The character set for MySQL/MariaDB database tables should be changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the statements that shorten table indexes, these must be run first. Beware that these upgrade statements may fail due to index collisions, which may occur when indexes are shortened. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
+The character set for MySQL/MariaDB database tables should be changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the statements that shorten table indexes, these must be run first. If you use DFS, see also `data/update/mysql/dbupdate-7.1.0-to-7.2.0-dfs.sql` for additional SQL statements. Beware that these upgrade statements may fail due to index collisions, which may occur when indexes are shortened. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
```

Comment:
AFAIR we can provide links to these files in this markdown so navigation is easier.